lib/service_base_url_test_kit/service_base_url_validate_group.rb

```diff
@@ -27,6 +27,31 @@ class ServiceBaseURLBundleTestGroup < Inferno::TestGroup
           ),
           optional: true
 
+    input :endpoint_availability_success_rate,
+          title: 'Endpoint Availability Success Rate',
+          description: %(
+            Select an option to choose how many Endpoints have to be available and send back a valid capability
+            statement for the Endpoint validation test to pass.
+          ),
+          type: 'radio',
+          options: {
+            list_options: [
+              {
+                label: 'All',
+                value: 'all'
+              },
+              {
+                label: 'At Least 1',
```

Comment:
```suggestion
                label: 'At Least One',
```

Just seems slightly more professional?